Gemfile

```diff
@@ -50,4 +50,8 @@ group :development do
   gem 'listen', '~> 3.0.0', :require => false
 end
 
+if File.exists? "#{__FILE__}.local"
```

Comment:
This would be more clear if it read `Gemfile.local` instead of being variable, since the name of the Gemfile will not change.